src/main/java/org/reactome/release/qa/check/HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck.java

```diff
@@ -14,7 +14,9 @@
  * Flags all human Reactions that contain non-disease PhysicalEntities that are non-human, or are human PhysicalEntities with relatedSpecies.
  */
 
-@SliceQACheck
+//NOTE: This QA test was turned off, as requested by Curation, in November 2020.
+//Uncomment below line to turn it back on.
```

Comment:
This suggests that there needs to be a better way of turning checks "off" (or "on"). Something that doesn't require the modification and recompilation of the project.

Reply:
Actually, now that you've said this, I believe there is an ExcludedChecks.txt file that has this exact purpose -- I just didn't think of it the other day. I'll make that change!